sdk/communication/communication-phone-numbers/test/public/utils/msUserAgentPolicy.ts

```diff
@@ -14,6 +14,8 @@ export function createMSUserAgentPolicy(): PipelinePolicy {
     name: "msUserAgentPolicy",
     async sendRequest(request: PipelineRequest, next: SendRequest): Promise<PipelineResponse> {
       const useragent = env.AZURE_USERAGENT_OVERRIDE;
+
+      request.headers.set("x-ms-useragent", "acs-test");
```

Comment:
I am not clear on this change, why are we setting default header value? Wouldn't it be better to just set environment variable AZURE_USERAGENT_OVERRIDE?
Btw. for live runs in pipeline, this variable is retrieved from kevault, and I believe it already has this value.

Reply:
This is only done for the tests and will not be affecting real usage of the SDK. This makes sure that every test that is run is marked as test traffic which helps us filter out the real traffic in monitoring.

I have also checked current execution of these tests in 2 pipelines and there is no evidence of x-ms-useragent and AZURE_USERAGENT_OVERRIDE being used. Here are logs:

phone numbers js pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3255434&view=logs&j=14c0e587-6dd4-5861-ebc6-f356f402a852&t=9c9f9821-1e35-5d4d-d26a-37127596262f
ts logs public: https://portal.microsoftgeneva.com/s/12937648
ts logs ppe: https://portal.microsoftgeneva.com/s/A06E70A6
ts logs int: 

phone numbers js weekly pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3269669&view=logs&j=14c0e587-6dd4-5861-ebc6-f356f402a852&t=9c9f9821-1e35-5d4d-d26a-37127596262f
ts logs public: https://portal.microsoftgeneva.com/s/1CEDB733
ts logs ppe: https://portal.microsoftgeneva.com/s/86408A45